hudi-common/src/main/java/org/apache/hudi/common/table/log/AbstractHoodieLogRecordReader.java

```diff
@@ -269,11 +274,6 @@ private void scanInternalV1(Option<KeySpec> keySpecOpt) {
           break;
         }
         if (logBlock.getBlockType() != CORRUPT_BLOCK && logBlock.getBlockType() != COMMAND_BLOCK) {
-          if (!completedInstantsTimeline.containsOrBeforeTimelineStarts(instantTime)
```

Comment:
Similar here wrt table version.  Before log file name change, this logic is still needed.